Tools/WinMLRunner/CommandLineArgs.h

```diff
@@ -90,4 +95,7 @@ class CommandLineArgs
     std::wstring m_inputData;
     std::wstring m_outputPath;
     uint32_t m_numIterations = 1;
+
+    float m_scaleFactor = 1.0f;
+    std::array<float, 3> m_meanStdDev{ 0, 0, 0 };
```

Comment:
m_meanStdDev corresponds to 3 floats, are these means or stddevs?? This name is quite confusing.

Reply:
These are actually precalculated factors that are accounting for the training dataset.  Here's a link to an example post from pytorch on how to pre-interpret the data.  I've seen that actual math that takes the means and standard deviation of the training data set to get these numbers, but haven't located it, yet.

Open to any suggestions here.

https://discuss.pytorch.org/t/how-to-preprocess-input-for-pre-trained-networks/683